packages/form/src/form/fields/ModelField/index.tsx

```diff
@@ -0,0 +1,6 @@
+import ModelField from './ModelField'
+
+export * from './VirtualizedAutocompleteList'
+export * from './ModelField'
+
+export default ModelField
```

Comment:
Don’t have to import and export again. Just do a direct export. Check how this is done in the other barrels